lib/gds-components/text-area/TextArea.tsx

```diff
@@ -42,7 +51,12 @@ export const TextArea = ({
         value={value}
         onChange={(e) => onChange(e.target.value)}
         id={id}
-        aria-describedby={characterLimit ? `${id}-character-count` : undefined}
+        aria-describedby={
+          cn({
+            [`${id}-character-count`]: !!characterLimit,
+            [`${id}-hint`]: !!hint,
+          }) || undefined
```

Comment:
I appreciate it's a bit weird to be using `classnames` here so I'm happy to change if people don't like it but I figured it does what we want 🤷

Reply:
I think a self-rolled version would look something like this:

```
const getAriaDescribedBy = (rules: { [ariaValue: string]: boolean }) =>
  Object.entries(rules)
    .filter(([, shouldApply]) => shouldApply)
    .map(([ariaValue]) => ariaValue)
    .join(" ") || undefined;
```

Note the `|| undefined` is so we don't default to empty string

Reply:
I've replaced anyway to avoid any future confusion around the use of `classnames` in this way